### DISCUSSION TITLE
Training with `torch.compile`

### DISCUSSION THREAD

**Discussion by Ramlaoui** [ACEsuit/mace#1000] [Q&A]
Hello,

I wondered if it were possible to fine-tune a MACE model using `torch.compile` with the `run_train` script. Do you have any option to do that? It seems like just naively using the default compile options doesn't work because of the gradient operation happening in the model? Is that the reason why the calculator cannot compute the stress if the model is compiled as well?

https://github.com/ACEsuit/mace/blob/b5faaa076c49778fc17493edfecebcabeb960155/mace/calculators/mace.py#L316-L323

Thank you!
